luigi/contrib/spark.py

```diff
@@ -122,6 +122,10 @@ def jars(self):
         return self._list_config(configuration.get_config().get("spark", "jars", None))
 
     @property
+    def packages(self):
+        return self._list_config(configuration.get_config().get("spark", "packages", None))
```

Comment:
Agreed Erik. I'll start working on a separate PR for refactoring the entire config of the spark class. In the mean-time this functionality is necessary so I think we should just merge this in.